### PR TITLE
[16.0][MIG] attachment_synchronize: Migration to 16.0

### PR DESCRIPTION
**Pull request by kevinkhao**
depends on https://github.com/OCA/server-tools/pull/2560

### CODE REVIEW

File: attachment_synchronize/tests/common.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    def _create_test_file(self):
+        self.backend.add(
+            os.path.join(self.directory_input, "bar.txt"),
+            self.filedata,
+            mimetype="text/plain",
+            binary=False,
+        )
+
+    def setUp(self):
+        super().setUp()
```
Comment (bealdav) [inline attachment_synchronize/tests/common.py:29]: Not sure but probably better to use `setUpClass(cls)` here ?

File: attachment_queue/models/attachment_queue.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            return False
+        else:
+            self.write(
+                {
+                    "state": "done",
+                    "date_done": fields.Datetime.now(),
+                    "running_lock": False,
+                }
+            )
+            return True
```
Comment (bealdav) [inline attachment_queue/models/attachment_queue.py:122]:
```suggestion
```

File: attachment_queue/models/attachment_queue.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                {"state": "failed", "state_message": str(e), "running_lock": False}
+            )
+            emails = self.failure_emails
+            if emails:
+                self.env.ref(
+                    "attachment_queue.attachment_failure_notification"
+                ).send_mail(self.id)
+            return False
+        else:
+            self.write(
```
Comment (bealdav) [inline attachment_queue/models/attachment_queue.py:115]:
```suggestion
            return self.write(
```

File: attachment_queue/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+* Valentin CHEMIERE <valentin.chemiere@akretion.com>
```
Comment (bealdav) [inline attachment_queue/readme/CONTRIBUTORS.rst:1]:
```suggestion
* Valentin CHEMIERE
```

**Review by bealdav — COMMENTED** [review]
some typo

**Comment by florian-dacosta** [comment]
Hi @kevinkhao I think you can close this one as the one with refactore has been merged some time ago

**Comment by kevinkhao** [comment]
> Hi @kevinkhao I think you can close this one as the one with refactore has been merged some time ago

Indeed, thanks

**Pull request closed without merging**